openwisp_monitoring/device/tests/test_models.py

```diff
@@ -578,6 +579,59 @@ def _create_env(self):
         )
         return dm, ping, load, process_count
 
+    def setUp(self):
+        from openwisp_monitoring.check.settings import CHECK_CLASSES
+
+        Check = load_model('check', 'Check')
+        self.device = self._create_device(organization=self._create_org())
+        self.ping_check = Check.objects.create(
+            name='Ping Check',
+            check_type=CHECK_CLASSES[0][0],
+            content_object=self.device,
+            params={},
+        )
+        self.device.monitoring.update_status('ok')
```

Comment:
The `setUp` method is executed before every test case, this is polluting the test envrironment and making other tests fail. We don't want to do this here. Use the `_create_env` method like we do in other tests to create the necessary objects.